template/my_component/frontend/src/MyComponent.tsx

```diff
@@ -1,84 +1,67 @@
 import {
   Streamlit,
-  StreamlitComponentBase,
   withStreamlitConnection,
+  ComponentProps,
 } from "streamlit-component-lib"
-import React, { ReactNode } from "react"
-
-interface State {
-  numClicks: number
-  isFocused: boolean
-}
+import React, { useEffect, useState, ReactElement } from "react"
 
 /**
  * This is a React-based component template. The `render()` function is called
```

Comment:
**suggestion:** Update this JSDoc comment since the `render()` function concept doesn't exist in function components

Reply:
Updated it to
```javascript
/**
 * This is a React-based component template. The passed props are coming from the 
 * Streamlit library. Your custom args can be accessed via the `args` props.
 */
```

What do you think?